rendermodules/montage/run_montage_job_for_section.py

```diff
@@ -167,8 +102,23 @@ def __init__(self, schema_type=None, *args, **kwargs):
         # Assigning the solver_executable to a different variable and removing it from args
         self.solver_executable = self.args['solver_executable']
         self.args.pop('solver_executable', None)
+        self.clone_section_stack = self.args['clone_section_stack']
+        self.args.pop('clone_section_stack')
+        if self.clone_section_stack:
+            if (self.args['first_section']!=self.args['last_section']):
+                raise ValidationError("first section and last section not equal")
 
     def run(self):
+
+        if self.clone_section_stack:
+            tmp_stack = "{}_z{}".format(self.args['source_collection']['stack'],self.args['first_section'])
```

Comment:
Maybe the temporary stack name should have a randomized or timestamped component -- in the case of reimaging and possible metadata errors z index is not reliable and it's possible that some of these stacks are not deleted after creation (e.g. in case of errors raised somewhere else in the script).

Reply:
done